From cf1e1aa8a37ea52f70874931b80611eee7a27f83 Mon Sep 17 00:00:00 2001
From: Andrew Eikum <aeikum@codeweavers.com>
Date: Mon, 26 Nov 2018 15:04:38 -0600
Subject: [PATCH 3/3] winepulse: Don't fake being one period behind in
 GetPosition
To: wine-devel <wine-devel@winehq.org>
Reply-To: wine-devel <wine-devel@winehq.org>,Andrew Eikum <aeikum@codeweavers.com>

---
 dlls/winepulse.drv/mmdevdrv.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/dlls/winepulse.drv/mmdevdrv.c b/dlls/winepulse.drv/mmdevdrv.c
index fb2e3678f8..39284e55ad 100644
--- a/dlls/winepulse.drv/mmdevdrv.c
+++ b/dlls/winepulse.drv/mmdevdrv.c
@@ -2769,13 +2769,6 @@ static HRESULT WINAPI AudioClock_GetPosition(IAudioClock *iface, UINT64 *pos,
 
     *pos = This->clock_written - This->held_bytes;
 
-    if(This->started){
-        if(*pos < This->period_bytes)
-            *pos = 0;
-        else if(This->held_bytes > This->period_bytes)
-            *pos -= This->period_bytes;
-    }
-
     if (This->share == AUDCLNT_SHAREMODE_EXCLUSIVE)
         *pos /= pa_frame_size(&This->ss);
 
-- 
2.19.2

